src/screens/ExplainSheet.js

```diff
@@ -158,8 +159,7 @@ const ENS_CONFIGURATION_EXPLAINER =
 export const explainers = (params, theme) => {
   const colors = theme?.colors;
   const chainId = params?.chainId;
-  const network = ethereumUtils.getNetworkFromChainId(chainId);
-  const networkName = chainIdToNameMapping[chainId];
+  const network = chainsName[chainId];
```

Comment:
with the changes below on line 540, you can now remove this line and the `chainsName` import